src/com/tutorial/main/Handler.java

```diff
@@ -1,51 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Graphics;
-import java.util.ArrayList;
-import java.util.LinkedList;
-
-public class Handler {
-	
-	LinkedList<GameObject> object = new LinkedList<GameObject>();
-	LinkedList<GameObject> toAdd = new LinkedList<GameObject>();
```

Comment:
And since these two objects are only used within `Handler` and never allocate a new list, they can be made `private final`.